spotbugs/src/main/java/edu/umd/cs/findbugs/ba/DefaultNullnessAnnotations.java

```diff
@@ -431,6 +431,10 @@ public static void addDefaultNullnessAnnotations(INullnessAnnotationDatabase dat
         database.addMethodAnnotation("java.util.Objects", "requireNonNull", "(Ljava/lang/Object;Ljava/lang/String;)Ljava/lang/Object;",
                 true, NullnessAnnotation.NONNULL);
 
+        database.addMethodParameterAnnotation("kotlin.jvm.internal.Intrinsics", "checkNotNullParameter", "(Ljava/lang/Object;)V",
+                true, 0, NullnessAnnotation.NONNULL);
+        database.addMethodParameterAnnotation("kotlin.jvm.internal.Intrinsics", "checkNotNullParameter", "(Ljava/lang/Object;Ljava/lang/String;)V",
+                true, 0, NullnessAnnotation.NONNULL);
```

Comment:
There are other similar nullness checking functions in [kotlin.jvm.internal.Intrinsics](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/runtime/kotlin/jvm/internal/Intrinsics.java). IMO it would be worth to add those as well.